README.md

```diff
@@ -283,6 +283,29 @@ before_action :check_locale
 
 Trying to figure out which key a certain translation on the page has? Append `?debug` to the url and `I18n.debug?` will be set - which leads to keys being printed on the page.
 
+### 🍪 Cookies
+
+Integrate cookie consent and add tracking capabilities such as Google Tag Manager and Google Analytics to your application with the following steps:
+
+**Include Shimmer's Consent Module**: Add the following line to your `application_controller.rb`:
+
+```ruby
+include Shimmer::Consent
+```
+
+**Add Google Tag Manager and Google Analytics**:
+
+*   If you wish to include Google Tag Manager and Google Analytics, insert the following lines to your `application.js`:
+
+```ruby
+ui.consent.enableGoogleTagManager("#{GOOGLE_TAG_MANAGER_ID}");
+ui.consent.enableGoogleAnalytics("#{GOOGLE_ANALYTICS_ID}");
```

Comment:
This is not Ruby:

```suggestion
```js
ui.consent.enableGoogleTagManager(GOOGLE_TAG_MANAGER_ID);
ui.consent.enableGoogleAnalytics(GOOGLE_ANALYTICS_ID);
```